Luminary099/R31.agc

```diff
@@ -212,11 +212,11 @@ GETRVN		CA	PRIO22		# INHIBIT SERVICER
 		TC	PRIOCHNG
```

Comment:
8 lines above, add a space after the tab (**T**ERMINATE aligns with **E** instead of **W**

![image](https://user-images.githubusercontent.com/3440094/95020904-d03c2c00-0665-11eb-9bf6-36af250e466d.png)